src/status_im2/common/home/banner/style.cljs

```diff
@@ -53,12 +54,12 @@
    {}))
 
 (defn banner-card-tabs-layer
-  [scroll-shared-value]
+  [scroll-shared-value testnet-enabled?]
   (reanimated/apply-animations-to-style
    {:transform [{:translate-y (animated-card-translation-y scroll-shared-value)}]}
    {:z-index  3
     :position :absolute
-    :top      (+ (safe-area/get-top) 192)
+    :top      (+ (safe-area/get-top) 192 (when testnet-enabled? constants/testnet-banner-height))
```

Comment:
@smohamedjavid - can we just put this in the safe area method directly? 🤔

Reply:
Thought of that too 😄 .

It will affect every component and screen that uses this. 

For example, This method is used in onboarding and login screens. We won't know whether the user has enabled test networks until he/she login.

Reply:
I guess we could consider the check _should show testnet banner_ based on whether testnet is active and the screen makes use of it?